### PR TITLE
[Composer] Added library necessary for Behat

### PR DESCRIPTION
**Pull request by Zales0123**

### CODE REVIEW

**Comment by pamil** [comment]
🚀 speed

**Comment by Zales0123** [comment]
I've added commit from https://github.com/Sylius/Sylius-Standard/pull/107 to avoid parallel errors.

File: .travis.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ before_script:
 
 script:
     - php -d opcache.enable_cli=0 bin/phpspec run --no-interaction -f dot
-    - bin/behat --strict -f progress -p cached || bin/behat --strict -f progress -p cached --rerun
+    - bin/behat bin/behat vendor/sylius/sylius/features/product/managing_products/select_taxons_for_product.feature:15 -p cached
```
Comment (pamil) [inline .travis.yml]: WTF

Comment (pamil) [inline .travis.yml]: bin/behat bin/behat xD

**Comment by Zales0123** [comment]
Closing in favor of https://github.com/Sylius/Sylius-Standard/pull/109 and https://github.com/Sylius/Sylius-Standard/pull/108

**Pull request closed without merging**